docs/en/install-upgrade/highlights.asciidoc

```diff
@@ -48,3 +49,18 @@ This list summarizes the most important enhancements in {kib} {minor-version}.
 include::{kib-repo-dir}/user/whats-new.asciidoc[tag=notable-highlights]
 
 :leveloffset: -1
+
+[[security-highlights]]
+=== {elastic-sec} highlights
+[subs="attributes"]
+++++
+<titleabbrev>{elastic-sec}</titleabbrev>
+++++
+
+This list summarizes the most important enhancements in {elastic-sec} {minor-version}.
```

Comment:
Do we want to change this so it says e.g. 8.0.0 instead of 8.0?

Reply:
@benironside Full X.y.z version seems OK to me, but all the other products use the shorter X.y version for their highlights pages. Here's the [landing page currently in public docs](https://www.elastic.co/guide/en/elastic-stack/8.0/elastic-stack-highlights.html), and you can click through to each product's page.

@jmikell821 WDYT?

Reply:
We'd like to discuss with other writers outside of Security, but we'll merge with current version to avoid blocking the rest of these changes.